src/main/java/com/ge/predix/log4j1/PredixLayoutPattern.java

```diff
@@ -41,11 +44,7 @@ protected String convert(final LoggingEvent event) {
         }
         logFormat.put("msg", event.getMessage());
         if (null != event.getThrowableInformation()) {
```

Comment:
I think the null check is important for non-error logs, where we don't want a stck property in the json at all. I can replace with a ternary operator, but the principle would remain the same.